### ISSUE TITLE
Is it possible to suppress this rule?

### ISSUE THREAD

**Issue by l3r8yJ** [volodya-lombrozo/jtcop#112]
https://github.com/volodya-lombrozo/jtcop/blob/0681a054880e4eac417351ff3e5526ad499f9ee5/src/main/java/com/github/lombrozo/testnames/rules/RuleAllTestsHaveProductionClass.java#L46

My test package has an annotation for the tests:
```java
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
@ActiveProfiles("test")
@SpringBootTest
public @interface TestWithSpringContext {
}
```

which is violation:
```bash
[ERROR] Failed to execute goal com.github.volodya-lombrozo:jtcop-maven-plugin:0.1.11:check (default) on project bot:
[ERROR] Test WithSpringContextTest.java doesn't have corresponding production class.
[ERROR] 	You can either rename or move the test class .../src/test/java/annotation/WithSpringContextTest.java:
[ERROR] 	More about that rule you can read here https://www.yegor256.com/2023/01/19/layout-of-tests.html#test-classes
```

I would like to add something like `@ForTestScope` that will do a pass or something like that

@volodya-lombrozo what do you think?

**Comment by l3r8yJ** [comment]
@volodya-lombrozo looks like in `0.1.12` it should be fixed, but it's still not work for annotations
```java
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
@ActiveProfiles("test")
@SpringBootTest
@SuppressWarnings("JTCOP.RuleAllTestsHaveProductionClass")
public @interface TestWithSpringContext {
}
```

**Comment by volodya-lombrozo** [comment]
@l3r8yJ I see... Ok, I will take a look. Apparently it's a problem with annotations and interfaces.

**Comment by volodya-lombrozo** [comment]
@l3r8yJ Could you check again, please? The bug should be fixed in the `0.1.13` version.

**Comment by l3r8yJ** [comment]
@volodya-lombrozo now it works, thanks!
